preprocessors/caption-recognition/caption.py

```diff
@@ -87,7 +87,8 @@ def captions():
     except jsonschema.exceptions.ValidationError as e:
         logging.error(e)
         return jsonify("Invalid Preprocessor JSON format"), 500
-    logging.debug(data)
+    logging.debug("Caption Recoginition Response")
```

Comment:
```suggestion
    logging.debug("Caption Recognition Response")
```

Reply:
For preprocessors that output information tightly related to the content, we have to be very careful what we log, for privacy reasons. It is fine to put things like categorizing content as a photo or chart, for example, but something like OCR would be a big problem, since we do not want to retain that information. Unless we're sure that the content of the response will not be a privacy issue, we need to remove this, or output something more abstract, but still potentially informative.